cloudvolume/__init__.py

```diff
@@ -1,3 +1,9 @@
+import sys
+
+if sys.version_info[0] >= 3:
+  import gevent.monkey
+  gevent.monkey.patch_all()
```

Comment:
Maybe add a warning about the monkey patching as it might affect other parts of their program and lead to weird issues.